grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -284,6 +284,23 @@ public GrpcServiceBuilder addServices(Iterable<BindableService> bindableServices
         return this;
     }
 
+    /**
+     * Adds gRPC {@link ServerServiceDefinition}s to this {@link GrpcServiceBuilder}.
+     */
+    public GrpcServiceBuilder addServiceDefinitions(ServerServiceDefinition... services) {
```

Comment:
To avoid same erasure error (due to `addService(BindableService)`), name this method `addServiceDefinitions`.
Any other better name?

Reply:
Can't think of any.. :thinking:

Reply:
🤔

Reply:
I'm fine with this. 🤔